federationsender/queue/queue.go

```diff
@@ -60,6 +60,13 @@ func NewOutgoingQueues(
 		signing:    signing,
 		queues:     map[gomatrixserverlib.ServerName]*destinationQueue{},
 	}
+	// Look up which servers we have pending items for and then rehydrate those queues.
+	if serverNames, err := db.GetPendingServerNames(context.TODO()); err == nil {
```

Comment:
`Background` probably more suitable at startup.

Reply:
We also drop the err to the floor?

Reply:
I'll switch to `Background`. Also I can log the error, although it doesn't really matter since another attempt to send a federation event would restart the queue anyway.

Reply:
Knowing that an error has occurred is useful especially for sqlite where `database is locked` is a constant worry.

Reply:
It's logging now.